recommonmark/parser.py

```diff
@@ -227,16 +228,30 @@ def inline_entity(inline):
     return entity_node
 
 
+def make_refnode(label, target, has_explicit_title):
```

Comment:
Kinda, it partially came from the `XRefRole` [here](https://github.com/sphinx-doc/sphinx/blob/master/sphinx/roles.py#L127), but just to understand how `pending_xref` worked. The goal is to make it work like the `:any:` role for references except when an url is given. I can add a comment about this.